src/Microsoft.DncEng.SecretManager/SecretTypes/AzureStorageTableSas.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.DncEng.SecretManager.SecretTypes
+{
+    [Name("azure-storage-table-sas-uri")]
+    public class AzureStorageTableSas : AzureStorageSasSecretType
```

Comment:
The name should probably have "Uri" in it, since it's not actually just the SAS (and some secrets might actually just be the SAS token themselves, which someone else concatenates later, if they are larger scoped)